src/Sylius/Behat/Page/Shop/Checkout/CompletePage.php

```diff
@@ -291,6 +297,25 @@ public function getShippingPromotionDiscount($promotionName)
     /**
      * {@inheritdoc}
      */
+    public function tryToOpen(array $urlParameters = [])
+    {
+        if ($this->getDriver() instanceof Selenium2Driver) {
+            $start = microtime(true);
+            $end = $start + 15;
+            do {
+                parent::tryToOpen($urlParameters);
+                sleep(1);
+            } while(!$this->isOpen() && microtime(true) < $end);
```

Comment:
That has no sense, if a page isn't open after one second, it will be reopened once again etc. so maximum loading time is always 1s but repeated 15 times